src/vmm/src/devices/virtio/net/device.rs

```diff
@@ -160,7 +161,8 @@ impl Net {
             | 1 << VIRTIO_NET_F_HOST_TSO4
             | 1 << VIRTIO_NET_F_HOST_UFO
             | 1 << VIRTIO_F_VERSION_1
-            | 1 << VIRTIO_RING_F_EVENT_IDX;
+            | 1 << VIRTIO_RING_F_EVENT_IDX
+            | 1 << VIRTIO_NET_F_GUEST_TSO6;
```

Comment:
nit: since we're adding IPv6 flags we might as well add `VIRTIO_NET_F_HOST_TSO6` and the `ECN` pair

Reply:
It seems out of scope of this CR, but if you are fine with that I will add in the next rev.